docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md

```diff
@@ -0,0 +1,94 @@
+---
+canonical: https://grafana.com/docs/grafana/latest/alerting/configure-notifications/manage-contact-points/integrations/configure-email/
+description: Configure the Email integration to connect alerts generated by Grafana Alerting
+keywords:
+  - grafana
+  - alerting
+  - email
+  - integration
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+menuTitle: Email
+title: Configure Email for Alerting
+weight: 300
+---
+
+## Configure Email for Alerting
+
+Use the Grafana Alerting - email integration to send email notifications when your alerts are firing. An email is sent when an alert fires and when an alert gets resolved.
+
+## Before you begin
+
+When utilizing Grafana OSS, enabling email notifications requires configuring [SMTP settings][smtp-settings] in the Grafana configuration settings. In contrast, for users of Grafana Cloud, SMTP configuration is not required.
+
+### SMTP configuration
+
+1. **Access the Configuration File.**
+
+   Locate the Grafana configuration file. This file is typically named `grafana.ini` or `custom.ini` and is located in the `conf` directory within the Grafana installation directory.
+
+1. **Open the Configuration File:**
+
+   Open the configuration file using a text editor.
+
+1. **Locate SMTP Settings Section.**
+
+   Search for the [SMTP settings section][smtp-settings] in the configuration file. It starts with `[smtp]`.
+
+1. **Configure SMTP Settings.**
```

Comment:
are these all screen elements? we only bold screen elements in this case

Reply:
nope they are not but i thought it increases readability.
i will "unbold" 😃  them